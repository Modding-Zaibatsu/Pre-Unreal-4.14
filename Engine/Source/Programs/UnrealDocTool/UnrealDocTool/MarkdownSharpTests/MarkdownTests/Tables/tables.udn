[Table 1 - Headers to top]
|             | Grouping                      ||
 First Header | Second Header  | Third Header  |
 ------------ | :------------: | ------------: |
 Content      |          *Long Cell*          ||
 Content      |      **Cell**  |         Cell  |
 New section  |     More       |         Data  |
     ^        |            And more           || 



[Table 2 - Headers to the right]
| ------------ | :------------: | ------------: |
|  Content      |          *Long Cell*          ||
|  Content      |      **Cell**  |         [INCLUDE:#List]  |
|  New section  |     More       |         Data  |
|    ^        |            And more      || 


<!--
[EXCERPT:List]
* Item 1
* Item 2
* Item 3
[/EXCERPT]
-->