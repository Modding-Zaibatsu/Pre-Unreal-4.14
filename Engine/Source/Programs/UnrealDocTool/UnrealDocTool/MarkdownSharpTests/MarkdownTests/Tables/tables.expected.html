<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
    <head>
    </head>
    <body>
        <table>
            <caption>Table 1 - Headers to top</caption>
            <col align=""/>
            <col align="center"/>
            <col align="right"/>
            <thead>
                <tr>
                    <th rowspan="1" colspan="1"/>
                    <th rowspan="1" colspan="2">
                            Grouping
                    </th>
                </tr>
                <tr>
                    <th rowspan="1" colspan="1">
                            First Header
                    </th>

                    <th rowspan="1" colspan="1">
                            Second Header
                    </th>
                    <th rowspan="1" colspan="1">
                            Third Header
                    </th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td rowspan="1" colspan="1" align="">
                            Content
                    </td>
                    <td rowspan="1" align="center" colspan="2">
                            <em>Long Cell</em>
                    </td>
                </tr>

                <tr>
                    <td rowspan="1" colspan="1" align="">
                            Content
                    </td>
                    <td rowspan="1" align="center" colspan="1">
                            <strong>Cell</strong>
                    </td>
                    <td rowspan="1" align="right" colspan="1">
                            Cell
                    </td>
                </tr>
                <tr>
                    <td rowspan="2" colspan="1" align="">
                            New section
                    </td>
                    <td rowspan="1" align="center" colspan="1">
                            More
                    </td>

                    <td rowspan="1" align="right" colspan="1">
                            Data
                    </td>
                </tr>
                <tr>
                    <td rowspan="1" align="center" colspan="2">
                            And more
                    </td>
                </tr>
            </tbody>
        </table>

        <table>
            <caption>Table 2 - Headers to the right</caption>
            <col align=""/>
            <col align="center"/>
            <col align="right"/>
            <tbody>
                <tr>
                    <th rowspan="1" colspan="1" align="">
                            Content
                    </th>
                    <td rowspan="1" align="center" colspan="2">
                            <em>Long Cell</em>
                    </td>
                </tr>
                <tr>
                    <th rowspan="1" colspan="1" align="">
                            Content
                    </th>
                    <td rowspan="1" align="center" colspan="1">
                            <strong>Cell</strong>
                    </td>
                    <td rowspan="1" align="right" colspan="1">
                        <ul >
                            <li>
                                <p>
                                    Item 1
                                </p>
                            </li>
                            <li>
                                <p>
                                    Item 2
                                </p>
                            </li>
                            <li>
                                <p>
                                    Item 3
                                </p>
                            </li>
                        </ul>
                    </td>
                </tr>
                <tr>
                    <th rowspan="2" align="" colspan="1">
                            New section
                    </th>
                    <td align="center" rowspan="1" colspan="1">
                            More
                    </td>

                    <td align="right" colspan="1" rowspan="1">
                            Data
                    </td>
                </tr>
                <tr>
                    <td align="center" rowspan="1" colspan="2">
                            And more
                    </td>
                </tr>
            </tbody>
        </table>    
    </body>
 </html>
